app/serializers/v2/people_serializer.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module V2
```

Comment:
maybe we can leverage inheritance here to avoid duplication in the serializer, this is a pattern we have adopted with the Allocation serializer

Reply:
Agree, that would be much cleaner 👍

Reply:
that's a good shout; I'll see if that can work here 👍

Reply:
Presumably this is just a rename of the old `PersonSerializer` ?

Reply:
To clarify, the original `person_serializer` was (temporarily) renamed `people_serializer`. It has now been renamed. back to `person_serializer`.

A new serializer, initially called `person_serializer` has been renamed `person_with_category_serializer` and it inherits from `person_serializer`.